hms-app/.env

```diff
@@ -1 +1 @@
-REACT_APP_API_ID=mxpub1nyld
\ No newline at end of file
+REACT_APP_CORE_URL=http://localhost:4566/restapis/${api_id}/local/_user_request_
```

Comment:
well you would still need to replace ${api_id} with something, and I meant it a bit differently but I didn't make it to clear.  However this should work perfectly until we get to the point where we have a production and local environment.  Then we would have either just environment variables or a function that will get the correct base url :)

Reply:
omg i replaced this after the commit -.-